src/sections/enterprise-offering/enterprise-bullets/enterprise-bullets.module.scss

```diff
@@ -0,0 +1,13 @@
+.benefits {
+	grid-gap: 40px;
+}
+.icon {
+	margin: 0 auto;
```

Comment:
the margin doesnt seem to do anything here

Reply:
you right.
done